03_AddUser.php

```diff
@@ -26,15 +26,14 @@
 		<div class="w3-top">
 			<div class="w3-bar w3-white w3-wide w3-padding w3-card">
 				<img src="images/placeholder.jpg" width="40" height="40">
-
 				<a href="index.html" class="w3-bar-item w3-button"><b>logo</b> </a>
 
-				<!-- Float links to the right. Hide them on small screens -->
-				<div class="w3-right w3-hide-small">	
```

Comment:
got rid of the 'hide screen' feature to ensure those with small screens are able to use the navbar